src/home/TabHome.tsx

```diff
@@ -74,13 +92,222 @@ function TabHome(_props: Props) {
     }
   }, [appState])
 
-  return <SafeAreaView testID="WalletHome" style={styles.container} edges={[]}></SafeAreaView>
+  const cKESToken = useCKES()
+  const cUSDToken = useCUSD()
+
+  function onPressAddCKES() {
+    AppAnalytics.track(TabHomeEvents.add_ckes)
+    if (cUSDToken?.balance.isZero()) {
+      !!cKESToken &&
+        navigate(Screens.FiatExchangeAmount, {
+          tokenId: cKESToken.tokenId,
+          flow: CICOFlow.CashIn,
+          tokenSymbol: cKESToken.symbol,
+        })
+    } else {
+      addCKESBottomSheetRef.current?.snapToIndex(0)
+    }
+  }
+
+  function onPressSendMoney() {
+    AppAnalytics.track(TabHomeEvents.send_money)
+    !!cKESToken &&
+      navigate(Screens.SendSelectRecipient, {
+        defaultTokenIdOverride: cKESToken.tokenId,
+      })
+  }
+
+  function onPressRecieveMoney() {
+    AppAnalytics.track(TabHomeEvents.receive_money)
+    navigate(Screens.QRNavigator, {
+      screen: Screens.QRCode,
+    })
+  }
+
+  function onPressHoldUSD() {
+    AppAnalytics.track(TabHomeEvents.hold_usd)
+    !!cKESToken &&
+      !!cUSDToken &&
+      navigate(Screens.SwapScreenWithBack, {
+        fromTokenId: cKESToken.tokenId,
+        toTokenId: cUSDToken.tokenId,
+      })
+  }
+
+  function onPressWithdraw() {
+    AppAnalytics.track(TabHomeEvents.withdraw)
+    navigate(Screens.WithdrawSpend)
+  }
+
+  return (
+    <SafeAreaView testID="WalletHome" style={styles.container} edges={[]}>
+      <FlatCard testID="FlatCard/AddCKES" onPress={onPressAddCKES}>
+        <View style={styles.column}>
+          {!!cKESToken && (
```

Comment:
I think we're doing this in quite a few places. We probably need some error state if tokens info is unavailable (e.g., if the network is down). Probably fine for now, but maybe required when we release the app. Should we make a placeholder ticket?

Reply:
I'm wondering if other error banners will be sufficient to let the user know that something is wrong and thats why the buttons aren't working. We'll have to experiment with trying to load the app when there is a network issue. I can create a ticket